README.md

```diff
@@ -186,3 +188,123 @@ spec:
 - Check that your application data has been restored:
 
 `oc get route <route-name> -n <app-ns> -ojsonpath="{.spec.host}"`
+
+<hr style="height:1px;border:none;color:#333;">
+
+<h4> Advanced Options <a id="Advanced-Options"></a></h4>
+
+The OADP VolumeSnapshotMover feature supports volume snapshots via the CSI driver. 
+Both CephRBD and CephFS are [supported via CSI](https://github.com/ceph/ceph-csi).
+The OADP VolumeSnapshotMover feature leverages some of the more recently added 
+features of Ceph and CSI to be performant in large scale environments.
+
+One of these newly added features for backups with CephFS to be more performant
+is the [shallow copy](https://github.com/ceph/ceph-csi/blob/devel/docs/design/proposals/cephfs-snapshot-shallow-ro-vol.md) 
+method, which is available > OCP 4.12.
+
+In large scale backups the OADP highly recommends OCP 4.12 and above. In OCP 4.12
+extra parameters on the DPA are required for this CephFS shallow copy.
+These parameters should not be required in OCP >= 4.13. 
+
+For backups on CSI backed by CephFS using shallow copy, OADP requires the 
+following volume options specified in the DPA.
+
+```
+volumeOptions:
+  sourceVolumeOptions:
+    accessMode: ReadOnlyMany
+    cacheAccessMode: ReadWriteMany
+    cacheStorageClassName: ocs-storagecluster-cephfs
+    moverSecurityContext: true
+    storageClassName: ocs-storagecluster-cephfs-shallow
```

Comment:
Shouldn't we also give an example of the custom ceph-fs storageclass spec with shallow copy enabled option ?